lib/api/core/plugins/pluginContext.js

```diff
@@ -238,8 +241,8 @@ function execute (kuzzle, request, callback) {
     return deferred;
   }
 
-  if (request.input.controller === 'realtime') {
-    error = new PluginImplementationError('Realtime controller is not available in plugins. You should use plugin hooks instead');
+  if (request.input.controller === 'realtime' && ['subscribe', 'unsubscribe'].includes(request.input.action)) {
```

Comment:
`Array.includes` won't work in Node.js 6

Reply:
https://node.green/#ES2016-features-Array-prototype-includes-Array-prototype-includes